dune-project

```diff
@@ -126,9 +126,9 @@ See more details on http://alt-ergo.ocamlpro.com/"
   (ocaml (>= 4.05.0))
   dune
   dune-configurator
-  (dolmen (>= 0.8))
-  (dolmen_type (>= 0.8))
-  (dolmen_loop (>= 0.8))
+  (dolmen (>= 0.8.1))
+  (dolmen_type (>= 0.8.1))
+  (dolmen_loop (>= 0.8.1))
```

Comment:
You should remove `(camlzip (>= 1.07))` from `alt-ergo-parsers` and add it to `alt-ergo-lib` since the calls to `camlzip` were moved to the library.